apps/mark-scan/backend/package.json

```diff
@@ -8,7 +8,7 @@
     "build"
   ],
   "scripts": {
-    "build": "tsc --build tsconfig.build.json && copyfiles -u 3 src/custom-paper-handler/fixtures/*.jpg build/custom-paper-handler/fixtures",
+    "build": "tsc --build tsconfig.build.json && copyfiles -u 3 src/custom-paper-handler/ballot-assets/*.* build/custom-paper-handler/ballot-assets",
```

Comment:
These ballot assets are required in prod and need a separate change to prod image build to ensure they make it there. We explicitly want to differentiate `fixtures` for dev from `ballot assets` for prod.

Reply:
I'm worried that it will be difficult to keep static ballot assets up to date as we make changes to code. It's a problem that's been a real annoyance in the past, requiring manual regeneration of the ballot assets (which is an easy place to make a mistake).

Is it possible to instead generate these assets automatically (e.g. on demand when used, or on application startup)? E.g. a blank page image data could be generated in memory, a ballot image could be rendered from an election def.